plugins/CoreAdminHome/tests/Integration/ConfigDeleteTest.php

```diff
@@ -0,0 +1,425 @@
+            '--section' => $sectionName,
+            '--key' => $settingName,
+            '-vvv' => false,
+        ];
+        if ( ! empty($value)) {
+            $inputArr['--value'] = $value;
+        }
+        $exitCode = $this->applicationTester->run($inputArr);
+
+        // Pass true to getDisplay(true) to normalize line endings, then trim() bc CLI adds an \ automatically.
+        $output = trim($this->applicationTester->getDisplay(true));
+        $debug && fwrite(STDERR, PHP_EOL . __FUNCTION__ . '::Got command output=' . $output) . PHP_EOL;
+
+        // Put the results in an easy-to-handle object format.
+        return (object) ['exitCode' => $exitCode, 'output' => $output];
+    }
+
+    private function runCommandWithArguments(string $sectionName, string $settingName = '', string $value = ''): object
+    {
+        $debug = false;
+        $debug && fwrite(STDERR, PHP_EOL . str_repeat('-', 80) . PHP_EOL . __FUNCTION__ . "::Started with \$sectionName=$sectionName; \$settingName=$settingName; \$value=$value") . PHP_EOL;
+
+        $inputArr = [
+            'command' => self::COMMAND,
+            '-vvv' => false,
+            'argument' => $sectionName . $settingName . (empty($value) ? '' : ".$value"),
+        ];
+        if ( ! empty($value)) {
+            $inputArr['--value'] = $value;
+        }
+        $exitCode = $this->applicationTester->run($inputArr);
+
+        // Pass true to getDisplay(true) to normalize line endings, then trim() bc CLI adds an \ automatically.
+        $output = trim($this->applicationTester->getDisplay(true));
+        $debug && fwrite(STDERR, PHP_EOL . __FUNCTION__ . '::Got command output=' . $output) . PHP_EOL;
+
+        // Put the results in an easy-to-handle object format.
+        return (object) ['exitCode' => $exitCode, 'output' => $output];
+    }
+
+    //
+    //*************************************************************************
+    // Tests that should yield errors.
+    //*************************************************************************
+    //
+    public function testNoArgsShouldYieldError()
+    {
+        $debug = false;
+
+        $inputArr = [
+            'command' => 'config:get',
+            '-vvv' => false,
+        ];
+        $exitCode = $this->applicationTester->run($inputArr);
+
+        // The CLI error code should be >0 indicating failure.
+        $this->assertGreaterThan(0, $exitCode);
+
+        // Pass true to getDisplay(true) to normalize line endings, then trim() bc CLI adds an \ automatically.
+        $output = trim($this->applicationTester->getDisplay(true));
+        $debug && fwrite(STDERR, PHP_EOL . __FUNCTION__ . '::Got command output=' . $output) . PHP_EOL;
+
+        $this->assertStringContainsString('InvalidArgumentException', $output);
+    }
+
+    public function testEmptyArgsShouldYieldError()
+    {
+        $debug = false;
+
+        // Pass empty section name.
+        $resultObj = $this->runCommandWithArguments('');
```

Comment:
Was it on purpose to only have one test running with arguments, that actually only passes an empty one? Maybe might be useful to also have some working tests and maybe some edge cases. (Same applies for the ConfigGetTest)

Reply:
They just fell off my radar.  Added tests with args to ConfigGetTest and ConfigDeleteTest classes.